submissions/AnnaStoyano/popup/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Popup</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+        <header>
+            <ul>
+               <li><a href="">Gmail</a></li>
+                <li><a href="">Images</a></li>
+                <li>
+                    <label for="main_menu" class="icon menu"></label>
+                    <input type="checkbox" id="main_menu"> 

```

Comment:
Watch the [video ](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide the checkbox correct way.